### PR TITLE
Rebase django-saml2-auth to 2.2.1

### PR DESCRIPTION
**Pull request by caioariede**

### CODE REVIEW

**Comment by FabioFleitas** [comment]
@caioariede Could we actually split this up into 2 PRs? One with the rebase and another with the single change for us? That way we can easily track via a PR?

**Comment by caioariede** [comment]
@FabioFleitas updated

File: README.rst [hunk 1]
```diff
@@ -119,10 +125,13 @@ How to use?
...
 
         SAML2_AUTH = {
-            # Required setting
+            # Metadata is required, choose either remote url or local file path
             'METADATA_AUTO_CONF_URL': '[The auto(dynamic) metadata configuration URL of SAML2]',
+            'METADATA_LOCAL_FILE_PATH': '[The metadata configuration file path]',
 
-            # Optional settings
+            # Optional settings below
+            'DEFAULT_NEXT_URL': '/admin',  # Custom target redirect URL after the user get logged in. Default to /admin if not set. This setting will be overwritten if you have parameter ?next= specificed in the login URL.
```
Comment (FabioFleitas) [inline README.rst:133]: Will we need to update this in our settings?

File: README.rst [hunk 2]
```diff
@@ -119,10 +125,13 @@ How to use?
...
+            'CREATE_USER': 'TRUE', # Create a new Django user when a new user logs in. Defaults to True.
```
Comment (FabioFleitas) [inline README.rst:134]: We will likely want to turn this off in our settings as well right?

**Review by FabioFleitas — APPROVED** [review]

**Pull request merged**